packages/runtime/test/lib/testUtils.ts

```diff
@@ -132,6 +132,32 @@ export async function syncUntilHasIdentityDeletionProcess(transportServices: Tra
     return await syncUntilHas(transportServices, "identityDeletionProcesses", (m) => m.id === identityDeletionProcessId.toString());
 }
 
+export async function resetEventBusAndSyncUntilHasEvent<TEvent extends Event>(
```

Comment:
still not a fan of this method. @Milena-Czierlinski can you try to find a cleaner solution?